### PR TITLE
Migrate Legacy CSS from Optly.git to OUI

### PR DESCRIPTION
**Pull request by daverau-optimizely**
# Background
Since @jamesopti ripped out Classic from Optimizely.git there was never a parallel effort to rip out legacy CSS from the app — ⚡️UNTIL NOW! ⚡️ 

This PR aims to migrate in-use CSS from Optimizely.git into OUI when it makes sense. Some of these are mixin files we should be using inside OUI, some of these are older patterns that are in use that never made it back into OUI, and a few are just helpful utilities that should live in OUI as the source of truth vs the monolith.

To be paired with this [Optimizely.git PR](https://github.com/optimizely/optimizely/pull/11272/) that removes over 7k lines of CSS debt from Classic and P13N, along with adding a few system-wide helpers from the app into OUI for better reuse.

## Migrate reusable helpers & mixins
- .border-dashed--all helper
- Pattern mixin for editor
- utilities (size, square, etc)
- no-select
- math functions
- lists
- keyframes

## Migrate style overrides
- Table defaults

### CODE REVIEW

File: src/oui/partials/sass/_mixins.scss [hunk 1]
```diff
@@ -2,6 +2,12 @@
 /// Mixins
 ////
 
+@import 'mixins/keyframes';
```
Comment (ayeshakmaz) [inline src/oui/partials/sass/_mixins.scss]: Loving this new mixin pattern!!

There are many form mixins in /oui/src/oui/partials/elements/_form.scss, should those also be separated out into a new file?

File: src/oui/partials/sass/mixins/_no-select.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@mixin no-select {
```
Comment (ayeshakmaz) [inline src/oui/partials/sass/mixins/_no-select.scss]: Might be good to add a comment above here explaining this mixin's purpose (similar to the others)

File: src/oui/partials/sass/_mixins.scss (hunk 1 above)
Comment (daverau-optimizely) [reply src/oui/partials/sass/_mixins.scss]: Those feel less like generic utilities that are reusable and more like input-specific CSS that we are also allowing to be consumed via mixin so it's easily extended. I would say not in this case.

File: src/oui/partials/sass/mixins/_no-select.scss (hunk 2 above)
Comment (daverau-optimizely) [reply src/oui/partials/sass/mixins/_no-select.scss]: Good call, did some cleanup on header comments.

**Pull request merged**